esp/scm/ecldirect.ecm

```diff
@@ -25,6 +25,8 @@ ESPStruct [nil_remove] ECLDirectException
     int LineNo;
     int Column;
     int Activity;
+    string Scope;
```

Comment:
should this be versioned?  The other changes in this file don't appear to have been.